experiment/posttest.json

```diff
@@ -0,0 +1,68 @@
+{

```

Comment:
Looks like the pretest and posttest have been taken from a sorting algorithm. The questions need to be changed.

Reply:
Yes, the animation team is working on it. They will get back with the all animations and questions soon.